RecoJets/JetPlusTracks/python/PATJetPlusTrackCorrections_cff.py

```diff
@@ -0,0 +1,93 @@
+import FWCore.ParameterSet.Config as cms
+# ---------- Add assigned jet-track association
+
+from RecoJets.JetAssociationProducers.trackExtrapolator_cfi import *
+trackExtrapolatorPAT = trackExtrapolator.clone(
+                      trackSrc = "produceTracks",
+                      trackQuality = 'highPurity'
+)
+
+from RecoJets.JetAssociationProducers.ak4JTA_cff import *
+ak4JetTracksAssociatorAtVertexJPTPAT = ak4JetTracksAssociatorAtVertex.clone(
+                                       useAssigned = True,
+                                       pvSrc = "offlineSlimmedPrimaryVertices",
+                                       jets = "slimmedCaloJets",
+                                       tracks = "produceTracks"
+)
+ak4JetTracksAssociatorAtCaloFaceJPTPAT = ak4JetTracksAssociatorAtCaloFace.clone(
+                                         jets = "slimmedCaloJets",
+                                         tracks = "produceTracks",
+                                         extrapolations = "trackExtrapolatorPAT"
+)
+ak4JetExtenderJPTPAT = ak4JetExtender.clone(
+                       jets = "slimmedCaloJets",
+                       jet2TracksAtCALO = "ak4JetTracksAssociatorAtCaloFaceJPTPAT",
+                       jet2TracksAtVX = "ak4JetTracksAssociatorAtVertexJPTPAT"
+)
+
+# ---------- Supported Modules
+
+from CommonTools.RecoAlgos.trackFromPackedCandidateProducer_cfi import *
+produceTracks = trackFromPackedCandidateProducer.clone()
```

Comment:
```suggestion
tracksFromPackedCandidates = trackFromPackedCandidateProducer.clone()
```
(a shorter ```tracksFromPacked``` or similar is OK)
IMHO this collection name is more intuitive than a rather generic "produceTracks", which also contains a verb which is rather uncommon.

Reply:
responding to https://github.com/cms-sw/cmssw/pull/30430/files#r459927238 here
```suggestion
produceTracks = trackFromPackedCandidateProducer.clone(PFCandidates = 'packedPFCandidates')
```
the PFCandidates is otherwise not initialized in the default fillDescriptions-made file.
This should work after removing the manually added trackFromPackedCandidateProducer_cfi